cornell-dti/course-plan
# Public Release - Removing Whitelist and Resolving Bugs

### Summary

This pull request removes the whitelist for CP (so CP is open to all Gmail accounts), addresses bugs after the data migration & push to prod for public release, and cleans up data migration scripts

- [x] Resolved bug where old courses had the same `uniqueId` as new courses. All courses should now have a distinct `uniqueId` (#446)
- [x] Removed data migration scripts
- [x] Removed whitelist and landingEmails 
- [x] Fixed bug where dupe semesters were added
- [x] Temp fix for creditRange to resolve broken coursemenu bug
- [x] Resolved outdated AS college data for Arts & Science users that breaks onboarding. Replaced outdated 'AS' with 'AS1' (#449)


### Test Plan
1.  You should be able to log in with a gmail not on the cp whitelist.
2. Try adding courses, adding/editing semesters, using onboarding.
2. Try adding a course like INFO 4998 and make sure that you can set a creditRange and you can access the course menu
3. Delete all your semesters, add a semester that's not Spring 2021. Then click Add for New Semester right after. You should not see dupes.
